src/ui/public/chrome/api/__tests__/xsrf.js

```diff
@@ -114,7 +113,7 @@ describe('chrome xsrf apis', function () {
       });
 
       it('accepts alternate tokens to use', function () {
```

Comment:
This is no longer a thing, correct? There's no custom token stuff going on behind the scenes since we're relying on version.  I'm actually mildly concerned that this didn't fail outright...